### ISSUE TITLE
SASS ampersand does not support

### ISSUE THREAD

**Issue by ilyalesik** [pranaygp/vscode-css-peek#54]
VS Code can't find class definition, created with ampersand at SASS file:
```sass
.some-block {
    width: 100%;
    height: 500px;

    &__description {
        margin: 22px 0;
    }
}
```
In this example, CSS Peek will find `.some-block` definition, but can't find `.some-block__description`.

**Comment by pucelle** [comment]
I wrote another extension based on CSS Peek, which supports ampersand reference. try CSS Navigation.

**Comment by yuriygumeniuk** [comment]
For SASS Long loaded and not working.

**Comment by pranaygp** [comment]
Fixed in 4.0.0+
